readme.txt

```diff
@@ -103,10 +103,34 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 == Changelog ==
 
+= 5.1 =
+
+* Release date: July 5th, 2017
+* Release post: https://jetpack.com/?p=20888
+
+**Enhancements**
+* Readers and commenters can now like each others blog post comments.

```

Comment:
New features like this have been put under a `Major Enhancements` header in prior releases